lib/rules/no-magic-numbers.js

```diff
@@ -90,6 +95,16 @@ module.exports = {
             return ignore.indexOf(value) !== -1;
         }
 
+        /**
+         * Returns whether the number should be ignore when used as a default
+         * value assignment.
+         * @param {ASTNode} parent the non-"UnaryExpression" parent.
+         * @returns {boolean} true if the number should be ignored
+         */
+        function isIgnoredDefaultValue(parent) {
```

Comment:
Can we pass `fullNumberNode` instead of its `parent`, for consistency with other functions? Also, we could additionally check whether it is its parent's `right`, for completeness.

Reply:
Fixed.